examples/game_of_life.cpp

```diff
@@ -66,9 +66,9 @@ int main()
     unsigned const height = 5;
 
     ddc::DiscreteDomain<DDimX> const domain_x
-            = ddc::init_trivial_space(ddc::DiscreteVector<DDimX>(length));
+            = ddc::init_trivial_bounded_space(ddc::DiscreteVector<DDimX>(length));
     ddc::DiscreteDomain<DDimY> const domain_y
-            = ddc::init_trivial_space(ddc::DiscreteVector<DDimY>(height));
+            = ddc::init_trivial_bounded_space(ddc::DiscreteVector<DDimY>(height));
```

Comment:
Why these are not the `constexpr`?

Reply:
The variables `length` and `height` are not intended to be `constexpr` but mimic a runtime parameter given by a user.